components/raftstore/src/router.rs

```diff
@@ -101,15 +98,15 @@ fn send_command_impl<EK, PR>(
     router: &PR,
     req: RaftCmdRequest,
     cb: Callback<EK::Snapshot>,
-    deadline: Option<Deadline>,
+    extra_opt: RaftCmdExtraOpts,
```

Comment:
should s/extra_opt/extra_opts.

Reply:
this is an one struct obj, should it be extra_opt?